src/main/java/pl/stormit/eduquiz/quizcreator/domain/category/Category.java

```diff
@@ -25,8 +28,11 @@ public class Category {
     @GeneratedValue
     private UUID id;
 
+    @NotBlank(message = "Category name can't be empty")
+    @Size(min =  2, max = 20)
     private String name;
 
     @OneToMany(mappedBy = "category", fetch = FetchType.EAGER)
+    @NotEmpty
```

Comment:
I don't know if there will be an exception here:
We have created a new category, but we don't have quizzes yet.